libmscore/score.h

```diff
@@ -18,6 +18,8 @@
  Definition of Score class.
 */
 
+#include <memory>
```

Comment:
really needed?

Reply:
It is necessary for std::shared_ptr. My class creates and returns a pointer to an object, and ownership of the object is transferred to another class. According to the code without std::shared_ptr this is not obvious. So that there are no problems with a memory leak or double deletion, in such cases it is convenient to use std::shared_ptr.

Reply:
OK. Not sure whether we'd want that added into all.h though

Reply:
It is in all.h, so not needed here